plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/DbResourceGroupConfigurationManager.java

```diff
@@ -129,7 +125,16 @@ public void destroy()
     public void start()
     {
         if (started.compareAndSet(false, true)) {
-            configExecutor.scheduleWithFixedDelay(this::load, 1, 1, TimeUnit.SECONDS);
+            Properties properties = new Properties();
```

Comment:
I'm not sure why you added `Properties` here. Why not updating `DbResourceGroupConfig`?